h2/src/main/org/h2/command/dml/Merge.java

```diff
@@ -33,10 +33,11 @@
  */
 public class Merge extends Prepared {
 
-    private Table table;
+    private Table targetTable;
```

Comment:
why does Merge change if you are creating a MergeUsing class?

Reply:
I renamed two fields and the getters/setters - to clarify their use - no other modifications made. 
As it happens - I learnt a lot from this class because I based the new merge in it - a great deal.